bootstrap/src/main/java/org/mobicents/media/server/bootstrap/configuration/XmlConfigurationLoader.java

```diff
@@ -82,8 +82,9 @@ public MediaServerConfiguration load(String filepath) {
             configureMedia(xml.configurationAt("media"), configuration.getMediaConfiguration());
             configureResource(xml.configurationAt("resources"), configuration.getResourcesConfiguration());
             configureDtls(xml.configurationAt("dtls"), configuration.getDtlsConfiguration());
-        } catch (ConfigurationException | IllegalArgumentException e) {
-            log.error("Could not load configuration from " + filepath + ". Using default values. Reason: " + e.getMessage());
+        } catch (Exception e) {
```

Comment:
Remove catch, no need to re-throw exception since we're catching it in upper layer.